src/languages/es.ts

```diff
@@ -1785,6 +1786,10 @@ export default {
             collect: 'Recolectar',
         },
         categories: {
+            deleteCategories: 'Eliminar categorías',
+            disableCategories: 'Desactivar categorías',
+            enableCategories: 'Activar categorías',
+            deleteFailureMessage: 'Se ha producido un error al intentar eliminar la categoría. Por favor, inténtalo más tarde.',
```

Comment:
Have the Spanish translations been verified?

Reply:
not yet, sending translation to open source channel

Reply:
please tag me so I can assist

Reply:
Seems like these translations were aprpoved!